epam_tr/src/test/java/com/epam/tr/service/logic/sorter/FileNameSorterTest.java

```diff
@@ -0,0 +1,24 @@
+package com.epam.tr.service.logic.sorter;
+
+import com.epam.tr.entities.FileEntity;
+import com.epam.tr.entities.FileType;
+import org.testng.annotations.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class FileNameSorterTest {
+
+    private final FileNameSorter sorter = new FileNameSorter();
+
+    @Test
+    public void testCompare() {
+        FileEntity firstEntity = new FileEntity(FileType.FILE, "name1", "path", 0);
+        FileEntity secondEntity = new FileEntity(FileType.FILE, "name2", "path", 0);
+        FileEntity thirdEntity = new FileEntity(FileType.FILE, "name2", "path", 0);
+        FileEntity fourthEntity = new FileEntity(FileType.FILE,"name","path",0);
+
+        assertEquals(-1,sorter.compare(firstEntity,secondEntity));
+        assertEquals(0,sorter.compare(secondEntity,thirdEntity));
+        assertEquals(1,sorter.compare(firstEntity,fourthEntity));
```

Comment:
you have 3 assets for different cases which look like a different tests.
So I'd write a test for each case.
Leave others Sorter tests as they are but fix this one.